model/user_test.go

```diff
@@ -118,29 +120,40 @@ func TestUserIsValid(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	user.FirstName = strings.Repeat("01234567890", 20)
-	if err := user.IsValid(); err == nil {
+	user.FirstName = strings.Repeat("a", 65)
+	if err := user.IsValid(); !HasExpectedError(err, "first_name", user.Id) {
 		t.Fatal(err)
 	}
 
-	user.FirstName = ""
-	user.LastName = strings.Repeat("01234567890", 20)
-	if err := user.IsValid(); err == nil {
+	user.FirstName = strings.Repeat("a", 64)
+	user.LastName = strings.Repeat("a", 65)
+	if err := user.IsValid(); !HasExpectedError(err, "last_name", user.Id) {
 		t.Fatal(err)
 	}
 
-	user.LastName = ""
-	user.Position = ""
+	user.LastName = strings.Repeat("a", 64)
+	user.Position = strings.Repeat("a", 64)
 	if err := user.IsValid(); err != nil {
 		t.Fatal(err)
 	}
 
-	user.Position = strings.Repeat("01234567890", 20)
-	if err := user.IsValid(); err == nil {
+	user.Position = strings.Repeat("a", 65)
+	if err := user.IsValid(); !HasExpectedError(err, "position", user.Id) {
 		t.Fatal(err)
 	}
 }
 
+func HasExpectedError(err *AppError, fieldName string, userId string) bool {
```

Comment:
Would you mind renaming this to something more specific to this function like `HasExpectedUserIsValidError`? I like  this function since we should really check for the correct error everywhere, but we'd need it to have a more unique name to use similar functions in other files